plugins/UM3NetworkPrinting/src/Messages/CloudFlowMessage.py

```diff
@@ -15,27 +15,23 @@
 
 class CloudFlowMessage(Message):
 
-    def __init__(self, address: str) -> None:
-
-        image_path = os.path.join(
-            CuraApplication.getInstance().getPluginRegistry().getPluginPath("UM3NetworkPrinting") or "",
-            "resources", "svg", "cloud-flow-start.svg"
-        )
+    def __init__(self, printer_name: str) -> None:
 
         super().__init__(
             text=I18N_CATALOG.i18nc("@info:status",
-                                    "Send and monitor print jobs from anywhere using your Ultimaker account."),
-            lifetime=0,
-            dismissable=True,
-            option_state=False,
-            image_source=QUrl.fromLocalFile(image_path),
-            image_caption=I18N_CATALOG.i18nc("@info:status Ultimaker Cloud should not be translated.",
-                                             "Connect to Ultimaker Digital Factory"),
+                                    f"Your printer <b>{printer_name}</b> could be connected via cloud.\n Manage your print queue and monitor your prints from anywhere connecting your printer to Digital Factory"),
```

Comment:
Does this f-string formatting work with our translations? If we export these translations, wouldn't it get evaluated to just an empty string or something? I'm not certain how this works.

Reply:
Good one. Is there an easy way to check this?

Reply:
I'm gonna do a quick test when component testing this, just be sure. If it doesn't work we should use the old method and see if we can fix it in our translation processing. Since that is on our todo list anyhow.

Reply:
Exporting the message results in the following entry:
```
msgctxt "@info:status"
msgid "Your printer <b>{printer_name}</b> could be connected via cloud.\n Manage your print queue and monitor your prints from anywhere connecting your printer to Digital Factory"
msgstr "Yer ship <b>{printer_name}</b> could be anchored via cloud.\n Manage yer scribble queue and monitor yer prints from anywhere boarding yer ship to Digital Factory"
```
We sometimes add the additional information for a translator that the string between the curly brackets doesn't need to be translated. But we don't do that anymore, looking in the history. Our translators do this correctly anyway.

I think it is safe to resolve this comment and merge the branch.